lib/github-package.js

```diff
@@ -585,11 +586,16 @@ export default class GithubPackage {
     this.setActiveContext(nextActiveContext);
   }
 
-  refreshAtomGitRepository(workdir) {
-    const atomGitRepo = this.project.getRepositories().find(repo => {
-      return repo && path.normalize(repo.getWorkingDirectory()) === workdir;
-    });
-    return atomGitRepo ? atomGitRepo.refreshStatus() : Promise.resolve();
+  async refreshAtomGitRepository(workdir) {
+    const directory = this.project.getDirectoryForProjectPath(workdir);
+    if (!directory) {
+      return;
+    }
+
+    const atomGitRepo = await this.project.repositoryForDirectory(directory);
+    if (atomGitRepo) {
+      await atomGitRepo.refreshStatus();
```

Comment:
So this doesn't actually prompt Atom's tree-view to recompute `workdir`'s directory icon to reflect that it's now a git repository. I did a little poking in tree-view source to see if that was possible and wasn't able to come up with anything.